test/pylib/manager_client.py

```diff
@@ -129,11 +142,19 @@ async def before_test(self, test_case_name: str) -> None:
 
     async def after_test(self, test_case_name: str, success: bool) -> None:
         """Tell harness this test finished"""
+        logging.getLogger('test').removeHandler(self.test_log_fh)
```

Comment:
You add the handler to root but remove from `test`, I'm guessing this call has no effect so we're leaking the handler?